metrics.json

```diff
@@ -0,0 +1,44 @@
+{
+  "last": 14400,
+  "sampling": 600,
```

Comment:
https://docs.sysdig.com/en/docs/developer-tools/working-with-the-data-api/#general-guidelines-for-using-the-data-api

Reply:
Can you move this into `config/` ?